2019-governance-committee-election.md

```diff
@@ -133,4 +134,88 @@ the nominee with the highest score after that will be picked.
 - October 23rd, Noon PT: Election end.
 - October 28th: Results are announced.
 
+## Appendix
+
+We identified active contributors. For these contributors we found at least one
+e-mail address associated with the commit. We will be using one of them at
```

Comment:
associated with the commit -> associated with the commits to OpenTelemetry repositories.